packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -223,7 +241,6 @@ const init = ({ mainWindow, store }: Dependencies) => {
             });
     });
 
-    // todo: why do we need that? this is called somewhere from react? settings? is it still relevant?
```

Comment:
This line was added in the previous commit and removed here. Can we not add it in the first place?